src/pages/wishlist/wishlistData.js

```diff
@@ -0,0 +1,48 @@
+import { v4 as uuid } from "uuid";
+export const wishData = [
+  {
```

Comment:
this seems static file, it should not be in pages folder. You can take to some other new folder say static or staticData

Reply:
The data is highly coupled with ui and i believe moving it around will introduce confusions.